docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   chrome:
     container_name: "edx.${COMPOSE_PROJECT_NAME:-devstack}.chrome"
     hostname: chrome.devstack.edx
-    image: edxops/chrome:${OPENEDX_RELEASE:-latest}
```

Comment:
What do you think about set `OPENEDX_RELEASE ?= olive.master` in `options.mk`? In many images it is repeated except for the images that we must take from paops.